ytfzf

```diff
@@ -32,6 +32,9 @@ tmp_video_json_file="/tmp/ytfzf-subjson"
 
 #enable/disable history
 enable_hist=${YTFZF_HIST-${enable_hist-1}}
+#enable/disable search history
+enable_search_hist=${YTFZF_SEARCH_HIST-${enable_search_hist-1}}
+enable_search_hist_menu=${YTFZF_SEARCH_HIST_MENU-${enable_search_hist_menu-1}}
```

Comment:
this should be 0 by default, when i ran it the first time it told me i had no search history, had to disable it